app/components/transUnit/TransUnitService.js

```diff
@@ -139,16 +114,12 @@
      */
     $rootScope.$on(EventService.EVENT.UNDO_EDIT,
       function (event, phrase) {
-        if (transUnitService.isTranslationModified(phrase)) {
-          if(phrase.plural) {
-            setAllTranslations(phrase, phrase.translations);
-          } else {
-            setTranslationText(phrase, phrase.translation);
-          }
+        if (PhraseUtil.hasTranslationChanged(phrase)) {
+          setAllTranslations(phrase, phrase.translations);
         }
       });
 
-    /**
+    /**run
      * EventService.EVENT.CANCEL_EDIT listener
      * Cancel edit and restore translation
      */
```

Comment:
What does 'run' refer to at the start of this comment? Is it just there by accident?